common/src/worldgen.rs

```diff
@@ -259,7 +265,7 @@ impl ChunkParams {
                 }
             } else if (slope_mod >= 3_f64) && (slope_mod <= 4_f64) {
                 voxel_mat = match voxel_mat {
-                    Material::Flowergrass => Material::Bigflowergrass,
+                    //Material::Flowergrass => Material::Bigflowergrass,
```

Comment:
Dead code?

Reply:
Is this alive enough to get merged? I could add some actual implementation for valley areas, but that feels like something for another branch.